### PR TITLE
Patch Ruby plugin bundling

### PR DESCRIPTION
**Pull request by bbergstrom**
Need to bundle ruby plugins with system bundler, _not_ embedded Chef bundler.  Also, fixing the test and making kitchen support aws credentials file.

### CODE REVIEW

File: definitions/bundle_install.rb [hunk 1]
```diff
@@ -1,16 +1,13 @@
 define :bundle_install do
-  # install bundler gem
-  gem_package 'bundler' do
-    options '--no-ri --no-rdoc'
-    action :upgrade
```
Comment (newzac) [inline definitions/bundle_install.rb]: No more upgrading and now we want to generate docs?

Comment (bbergstrom) [inline definitions/bundle_install.rb]: The forced upgrade is not really necessary, just need bundler to be installed.
I'd rather have simpler code here than save a few seconds on install that will happen once per server, so I removed the options.

Comment (newzac) [inline definitions/bundle_install.rb]: Makes sense to me.

**Comment by newzac** [comment]
CR 👍

**Pull request merged**